<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>focusController</class>
 <widget class="QMainWindow" name="focusController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>279</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelComport">
      <property name="text">
       <string>COM PORT</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelComport_2">
      <property name="text">
       <string>Connection PORT</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditComport">
      <property name="text">
       <string>COM5</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="2">
     <widget class="QPushButton" name="buttonSetPorts">
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="moveButton1">
      <property name="text">
       <string>Move</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="moveSpinbox1">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>6000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="homeButton">
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="zeroButton">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777209</height>
       </size>
      </property>
      <property name="text">
       <string>Zero</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QDoubleSpinBox" name="incSpinbox">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>0.100000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="moveButton2">
      <property name="text">
       <string>Move</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="moveButton3">
      <property name="text">
       <string>Move</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDoubleSpinBox" name="moveSpinbox3">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>6000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="downButton">
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditConnport">
      <property name="text">
       <string>10125</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="digitCount">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="moveSpinbox2">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>6000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="upButton">
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>248</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuRemote">
    <property name="title">
     <string>&amp;Remote</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRemote"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Connect">
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Disconnect">
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
